.gitignore

```diff
@@ -13,3 +13,5 @@ native/native_main
 native_main.spec
 .wine-pyinstaller
 tags
+compiler/gen_metadata.js
+src/metadata.ts
```

Comment:
Can you follow convention established by other generated typescript and use .blah_generated.ts?

Reply:
Unfortunately we're not allowed to choose the name of compiler/gen_metadata.js because we're compiling to commonjs. I'm not sure what compiling to commonjs is or does but I had trouble using other module systems (nodejs doesn't seem to be able to import amd/system modules).
Moving metadata.ts is fine though.

Reply:
That's fine, just wanted to keep src directory a bit tidy : )